staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_tls_test.go

```diff
@@ -28,31 +28,103 @@ import (
 	utiltesting "k8s.io/client-go/util/testing"
 )
 
+var rsaSelfSignedCertPEM = `
+-----BEGIN CERTIFICATE-----
+MIIDZTCCAk2gAwIBAgIUEARJE682DlpLrwXr32hkJo2OHOowDQYJKoZIhvcNAQEL
+BQAwQjELMAkGA1UEBhMCVVMxFTATBgNVBAcMDERlZmF1bHQgQ2l0eTEcMBoGA1UE
+CgwTRGVmYXVsdCBDb21wYW55IEx0ZDAeFw0yMTA0MTIxMTEyMDlaFw0yMTA1MTIx
+MTEyMDlaMEIxCzAJBgNVBAYTAlVTMRUwEwYDVQQHDAxEZWZhdWx0IENpdHkxHDAa
+BgNVBAoME0RlZmF1bHQgQ29tcGFueSBMdGQwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+DwAwggEKAoIBAQDQOIOlz+GhLxwigsBBj6ZXOB6DNK9DACmmw0pz3M+U0o4+PI85
+8ae3q2eizvjMwCHgvQmh82w9kaI2NehnXCygG4qi7MTRNj+UnsrP5haTc5FyucYl
+GUADD9MUuyR9qZwkAt+PY4QmRotWnBlKLD/I+rXBVVv1KveJUkxoBLGk42kpMdS7
+RT06vmpGVHjq9HikrRvicdFbUfm4YODvFMNNStnoInZJmmGxumnGxhNkO+n6mswk
+3/Je5QEuZ8S2yIGkMXVOCUzAeScbI+NGiursYx5OPjN0doR4xNEHYIC53ATDBaK3
+z3Hxhp2tYNPDbZvGnFPsjcFAiXspYjViQDVJAgMBAAGjUzBRMB0GA1UdDgQWBBTR
+a1tRtnbp9ZQruY5RSvmzP/duiTAfBgNVHSMEGDAWgBTRa1tRtnbp9ZQruY5RSvmz
+P/duiTAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQCYtfHoz7Y8
+evmSGUqfOay7DCyL2tFLdWJdki4YG0NDquA9QKZ8jfl0epUnlX+UaaxT1fswAd0H
+YO0V48MCADlx2xC/pDmSzIS8chPbipqpGZuHMl+LseRaltZbnyMf7VSrK0EbW0xh
+bKbZDPR3lSkAQaCmYqlzauY0ZJa3bHZrhzem0wMUdwTFH03AJASqq3PzlGEOcuqa
+6Z0me1WVcR9oHwfbfOEF3DiQinFhRyG/DtCD2oYbbaO9e9VP0+1Hy05YkTN572Gw
+9jF5Z4wn5rFZJNglVoDiTEPwUyt+iXdGRPvQ7ftaTmK+jfbwxNbMMjehOi2y1nCW
+GIvEgkp0W7eG
+-----END CERTIFICATE-----
+`
+
 var rsaCertPEM = `-----BEGIN CERTIFICATE-----
-MIIB0zCCAX2gAwIBAgIJAI/M7BYjwB+uMA0GCSqGSIb3DQEBBQUAMEUxCzAJBgNV
-BAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBX
-aWRnaXRzIFB0eSBMdGQwHhcNMTIwOTEyMjE1MjAyWhcNMTUwOTEyMjE1MjAyWjBF
-MQswCQYDVQQGEwJBVTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50
-ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMFwwDQYJKoZIhvcNAQEBBQADSwAwSAJBANLJ
-hPHhITqQbPklG3ibCVxwGMRfp/v4XqhfdQHdcVfHap6NQ5Wok/4xIA+ui35/MmNa
-rtNuC+BdZ1tMuVCPFZcCAwEAAaNQME4wHQYDVR0OBBYEFJvKs8RfJaXTH08W+SGv
-zQyKn0H8MB8GA1UdIwQYMBaAFJvKs8RfJaXTH08W+SGvzQyKn0H8MAwGA1UdEwQF
-MAMBAf8wDQYJKoZIhvcNAQEFBQADQQBJlffJHybjDGxRMqaRmDhX0+6v02TUKZsW
-r5QuVbpQhH6u+0UgcW0jp9QwpxoPTLTWGXEWBBBurxFwiCBhkQ+V
+MIIDCzCCAfMCFAIsO+psszOfir2J0i6dqWN6RnBQMA0GCSqGSIb3DQEBCwUAMEIx
```

Comment:
Probably after removing the CA path validation from tlsCert and key, the tests needs to be rewriten / rolled back :)

Reply:
Nah, I regenerated the certificates, etc, to have a valid CA.  It's not tested (now), but it's not wrong to have it in.  I can of course revert that part if we don't issue a warning (see above) if you want.

Reply:
yup, if the former works fine for us it might be better to revert, so someone looking into this PR in the future wont get confused by 'why this changed and is a no-op?' :D

Reply:
Changed in the last patch.